packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -69,7 +69,7 @@ class DDSLegalNav extends StableSelectorMixin(LitElement) {
 
   render() {
     return html`
-      <nav class="${prefix}--legal-nav">
+      <nav class="${prefix}--legal-nav" aria-label="Legal Navigation">
```

Comment:
This is good, but it would be better by allowing the user to set this, mainly to accommodate for translations. You could make a property to make this easier for the adopter.

Reply:
Added the aria label as property. 
Thanks